test/enhancedTreeFormatter.test.ts

```
@@ -0,0 +1,227 @@
+import { test, expect } from 'bun:test';
+import { formatAsEnhancedTree } from '../src/formatters/enhancedTreeFormatter.js';
+import type { Node, ScanResult, TreeOptions } from '../src/types/index.js';
+import { AnalysisMode, TreeSortBy, MetricType } from '../src/types/index.js';
```

ESLint

[error] 'MetricType' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)